.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "callstack-io",
+  "rules": {
+    "global-require": 0,
+    "flowtype/no-weak-types": 0
+  }
+}
```

Comment:
Also this file should be named `.eslintrc.json`. Config files without extension are now deprecated in ESLint